files/en-us/web/accessibility/aria/aria_techniques/using_the_alert_role/index.html

```diff
@@ -115,7 +115,7 @@ <h3 id="Notes">Notes </h3>
 <ul>
  <li>Using the alert role on an element implies that element has <code>aria-live="assertive"</code>.</li>
  <li>The alert role should only be used for static text content. The element that the alert role is used on does not have to be able to receive focus, as screen readers will automatically announce the alert regardless of where keyboard focus is currently located.</li>
- <li>If an alert also provides interactive controls (such as form controls that allow the user to rectify a problem, or an "OK" button that discards the alert) the <a href="/en/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role" title="Using the alert role">alertdialog</a> role should be used instead. </li>
+ <li>If an alert also provides interactive controls (such as form controls that allow the user to rectify a problem, or an "OK" button that discards the alert) the <a href="/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role" title="Using the alert role">alertdialog</a> role should be used instead. </li>
```

Comment:
Remove this `title` too.
```suggestion
 <li>If an alert also provides interactive controls (such as form controls that allow the user to rectify a problem, or an "OK" button that discards the alert) the <a href="/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role">alertdialog</a> role should be used instead. </li>
```

Reply:
Same as [below](https://github.com/mdn/content/pull/722#discussion_r550339225), this `title` adds extra information to the contents.